### PR TITLE
More Ian Hats

### PR DESCRIPTION
**Pull request by Dorsisdwarf**
ian
What does Ian really need? More hats!
Now you can have your own little Dark Lord as you march the halls in an unstoppable wave of death.

Also makes regular corgis look wistfully at food more often, because it's adorable.

### CODE REVIEW

File: NTstation13.dme [hunk 1]
```diff
@@ -951,6 +951,7 @@
 #include "code\modules\mob\living\carbon\alien\larva\inventory.dm"
 #include "code\modules\mob\living\carbon\alien\larva\larva.dm"
 #include "code\modules\mob\living\carbon\alien\larva\life.dm"
+#include "code\modules\mob\living\carbon\alien\larva\login.dm"
```
Comment (RemieRichards) [inline NTstation13.dme]: Untick this file

File: NTstation13.dme [hunk 2]
```diff
@@ -960,6 +961,7 @@
 #include "code\modules\mob\living\carbon\brain\death.dm"
 #include "code\modules\mob\living\carbon\brain\emote.dm"
 #include "code\modules\mob\living\carbon\brain\life.dm"
+#include "code\modules\mob\living\carbon\brain\login.dm"
```
Comment (RemieRichards) [inline NTstation13.dme]: Untick this file

File: NTstation13.dme [hunk 3]
```diff
@@ -1236,6 +1238,7 @@
 #include "code\modules\surgery\implant_removal.dm"
 #include "code\modules\surgery\limb augmentation.dm"
 #include "code\modules\surgery\lipoplasty.dm"
+#include "code\modules\surgery\multiple_location_example.dm"
```
Comment (RemieRichards) [inline NTstation13.dme]: Untick this file

**Comment by Dorsisdwarf** [comment]
Yeah, I will. DreamMaker unticked all the files and I had to retick them from memory.

**Comment by RemieRichards** [comment]
Fair enough, Those files aren't harmful to the game, just not needed.

For example, the 2 login files are login procs that do NOTHING extra.
and the multiple loc example file is just pure documentation.

**Comment by Dorsisdwarf** [comment]
Git is being an arse and refusing to push, giving me "Failed to exit cleanly, exit code 1"

**Comment by RemieRichards** [comment]
edit the dme manually on the PR itself ON github.

**Comment by Dorsisdwarf** [comment]
Done.

**Comment by RemieRichards** [comment]
The blank lines are there but they'll be removed the next time someone commits a new file, so were good.

**Pull request merged**